### DISCUSSION TITLE
How to create profile readme

### DISCUSSION THREAD

**Discussion by me-masoud** [community/community#38671] [Other Feature Feedback, Questions, & Ideas]
I have a problem on creating  profile readme when I go through steps in [this document](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/managing-your-profile-readme) I get error on creating a repository with name same as my username.
I get this error :
 Name has been retired and cannot be reused  
 How can i fix this problem  ?

**Comment by ossanna16** [comment]
Hi there @masoud-me and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
